homeassistant/components/konnected/binary_sensor.py

```diff
@@ -72,6 +74,13 @@ def device_class(self):
         """Return the device class."""
         return self._device_class
 
+    @property
+    def device_info(self):
+        """Return the device info."""
+        return {
+            "identifiers": {(KONNECTED_DOMAIN, self._device_id)},
+        }
+
     async def async_added_to_hass(self):
         """Store entity_id and register state change callback."""
         self._data[ATTR_ENTITY_ID] = self.entity_id
```

Comment:
Where is this used?

Reply:
It's existence is checked in __init__.KonnectedView.get to determine if an entity associated with a config has been created.

Reply:
Ok. It's ok to keep it too, I think.